app.rb

```diff
@@ -3,7 +3,9 @@
 require_relative 'student'
 require_relative 'teacher'
 require_relative 'rental'
+require 'json'
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
- [ ] Hi, you have done a good job so far, I would advise you to solve the Linter manually. Why:
- A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).

There are a few reasons for using linters:

- Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
- The entire codebase looks like written by one person.
- Programmers can focus on solving problems, instead of cleaning up the code.